packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -0,0 +1,112 @@
+import { Component, h, JSX, Listen, Element, Prop, State } from '@stencil/core';
+
+import { trackComponent } from '@utils/tracking/usage';
+import {
+  validateFormIds,
+  getSlottedInput
+} from '../../../stable/gux-form-field/gux-form-field.service';
+import { GuxSelectorCardVariant } from './gux-selector-card.types';
+import { calculateInputDisabledState } from '@utils/dom/calculate-input-disabled-state';
+import { onInputDisabledStateChange } from '@utils/dom/on-input-disabled-state-change';
+import { preventBrowserValidationStyling } from '@utils/dom/prevent-browser-validation-styling';
+
+@Component({
+  styleUrl: 'gux-selector-card.scss',
+  tag: 'gux-selector-card',
+  shadow: true
+})
+export class GuxSelectorCard {
+  private input: HTMLInputElement;
+  private disabledObserver: MutationObserver;
+
+  @Element()
+  private root: HTMLElement;
+
+  @Prop()
+  variant: GuxSelectorCardVariant = 'simple';
```

Comment:
Does this property do anything ?

Reply:
From what I see in figma if its `descriptive` it will be left aligned and have a description area.

Reply:
I decided to just include the simple card styling at this point and leave the descriptive variant until a later card just to keep the PR smaller.